src/pbrt/film.cpp

```diff
@@ -78,8 +79,16 @@ FilmBaseParameters::FilmBaseParameters(const ParameterDictionary &parameters,
     } else if (filename.empty())
         filename = "pbrt.exr";
 
-    fullResolution = Point2i(parameters.GetOneInt("xresolution", 1280),
-                             parameters.GetOneInt("yresolution", 720));
+    if (Options->fullscreen) {
+        fullResolution = GUI::GetResolution();
+
+        // Omit unused parameter error
+        auto unusedX = parameters.GetOneInt("xresolution", 1280);
+        auto unusedY = parameters.GetOneInt("yresolution", 720);
+    } else {
+        fullResolution = Point2i(parameters.GetOneInt("xresolution", 1280),
```

Comment:
When in `Options->quickRender` mode we probably only want to scale the resolution by a 1/4 when not in fullscreen mode.

Reply:
Also automatically setting sampling to 1 is probably not what you want in interactive/fullscreen mode. The simplest would be to just not allow quick rendering in interactive/fullscreen mode (or getting rid of it since it is from an era where rendering took hours). I'm happy to implement anything if Matt/you come to a conclusion.